src/plugins/intel_cpu/src/nodes/tensoriterator.cpp

```diff
@@ -521,6 +653,8 @@ void TensorIterator::execute(dnnl::stream strm) {
         // copy data to subgraph iteration
         for (auto &mapper : before_mappers)
             mapper->execute(strm, i);
+        for (auto &mapper : back_mappers)
+            mapper->execute(strm, i);
```

Comment:
A minor problem here: the inputs of `MergedInput` type are copied twice in the first iteration. The first time is in first_mappers execute, and the second time is here in back_mappers, as these inputs are used to initialized in both mappers.